v21.1/connect-to-the-database.md

```diff
@@ -6,7 +6,7 @@ toc: true
 
 <div class="filters filters-big clearfix">
   <button class="filter-button current">Local</button>
-  <a href="connect-to-the-database-cockroachcloud.html"><button class="filter-button"{{ site.data.products.dedicated }}
+  <a href="connect-to-the-database-cockroachcloud.html"><button class="filter-button">{{ site.data.products.dedicated }}
```

Comment:
```suggestion
  <a href="connect-to-the-database-cockroachcloud.html"><button class="filter-button">{{ site.data.products.dedicated }}</button></a>
```

Reply:
whoops thank you! and sorry for all the pushes